jar-infer/jar-infer-lib/src/test/java/com/uber/nullaway/jarinfer/JarInferTest.java

```diff
@@ -65,6 +65,7 @@
    * A dummy checker to allow us to use {@link CompilationTestHelper} to compile Java code for
    * testing, as it requires a {@link BugChecker} to run.
    */
+  @SuppressWarnings("BugPatternNaming") // remove once we require EP 2.11+
```

Comment:
Is this complaining about this being an inner class? As far as I can tell, `DummyChecker` matches both the name of the class and of the checker.

Also, separately, should that `@BugPattern` annotation not be under the javadoc rather than above it?

Reply:
So the complaint in EP 2.11.0 is that we set the `name` parameter of the `@BugPattern` annotation to the same name as the class, which is redundant as of EP 2.11.0.  The problem is that this name parameter is still required for EP versions before 2.11.0.  So we can't fix this without losing compatibility with older EP versions.

I shifted the annotation position in [5c1890c](https://github.com/uber/NullAway/pull/567/commits/5c1890cdcd37a9d75c41d0a07574911f55e04e06)